components/common/header/authNav.js

```diff
@@ -114,8 +118,10 @@ const AuthNav = ({ user, onLogout }) => {
             >
               <Link href='/how-it-works'>
                 <a>
-                  <Icon name='info' marginRight='.5rem' />
-                  How It Works
+                  <Box as='span' display='flex' alignItems='center'>
+                    <Icon name='info' marginRight='.5rem' />
+                    <span>How It Works</span>
```

Comment:
i think we made these lower case after initial word

Reply:
Done!